frameworks/hdfs/universe/config.json

```diff
@@ -35,6 +35,11 @@
           "type":"string",
           "default":"svc.yml"
         },
+        "tls_enabled": {
```

Comment:
Can you split this up into `{"tls": {"enabled": ...}}`.  For 1) consistency with `kerberos`, and 2) forwards compatibility with future TLS configuration variables.